### PR TITLE
Feeture/pagination untwig

### PR DESCRIPTION
**Pull request by msawicki**
## What does this do/fix?

Adds a single pagination "component"

### CODE REVIEW

File: wp-content/themes/core/components/pagination/single/single.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?php
+// phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+$c = \Tribe\Project\Templates\Components\pagination\single\Controller::factory( $args );
+
+?>
+<?php if ( ! empty( $c->previous_post ) || ! empty( $c->next_post ) ) { ?>
+	<nav <?php echo $c->wrapper_classes(); ?><?php echo $c->wrapper_attrs(); ?> >
+
+		<h3 <?php echo $c->header_attrs(); ?><?php echo $c->header_classes(); ?>>
```
Comment (msawicki) [inline wp-content/themes/core/components/pagination/single/single.php:9]: I kind of made this component up, do we want any tag customization on the header or the wrapper?

File: wp-content/themes/core/components/pagination/single/single.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+			<?php esc_html_e( 'Post Pagination', 'tribe' ); ?>
+		</h3>
+
+		<ol <?php echo $c->container_classes(); ?><?php echo $c->container_attrs(); ?>>
```
Comment (msawicki) [inline wp-content/themes/core/components/pagination/single/single.php:13]: Will this always be ol? Do we need an option where this is an ul?

Comment (ryanurban) [reply wp-content/themes/core/components/pagination/single/single.php:13]: Nope, will always be ol

File: wp-content/themes/core/components/pagination/single/single.php (hunk 1 above)
Comment (ryanurban) [reply wp-content/themes/core/components/pagination/single/single.php:9]: Nope, and I'll actually nix the `h3` in favor of adding a `aria-label` and putting the static header text as the value

**Pull request merged**